src/panels/awsConfig.ts

```diff
@@ -1,15 +1,18 @@
 import { UxSDKAwsConfig } from 'aws-iot-twinmaker-grafana-utils';
 import { Credentials } from 'aws-sdk/global';
+import { CredentialProvider } from '@aws-sdk/types';
 
 const defaultRegion = 'us-east-1';
 
 export const getAwsConfig = (
   credentialsProvider: () => Promise<Credentials>,
+  credentialsProviderV3: CredentialProvider,
   updatedRegion?: string
 ): UxSDKAwsConfig => {
   const region = updatedRegion ? updatedRegion : defaultRegion;
   return {
     credentialsProvider,
+    credentialsProviderV3,
```

Comment:
I don't see where this is used

Reply:
This is used by the updated grafana-utils to call S3 in the SceneViewer and KVS in the VideoPlayer. This was a recent update that uses both AWS SDK V2 and V3.